content/03-reference/01-tools-and-interfaces/02-prisma-client/08-pagination/index.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: 'Pagination'
+metaTitle: 'Pagination (Reference)'
+metaDescription: 'Prisma Client provides features that are not achievable with relational databases. These features are referred to as "polyfills" and explained on this page.'
+---
+
+## Overview
+
+Prisma Client supports both offset and cursor-based pagination.
+
+## Offset pagination
+
+Offset pagination combines `skip` and `take` to skip a certain number of results and select a limited range. The following query skips the first 3 `Post` records and returns records 4 - 7:
```

Comment:
nit: "combine" on concept that were not introduced before felt weird to me

Reply:
Fair - changed to 'uses' in both instances.